src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md

```diff
@@ -1,35 +1,48 @@
 
-System administrators can use the CLI to set Ecosystem-wide configuration properties after Ecosystem installation.
+Integration testing can be complicated. Tests often require configuration parameters to bind to a specific test environment. Galasa Managers require configuration so that systems under test can be contacted. Credentials for systems need to be supplied to tests and Managers to enable connections. The organisation of these parameters must be centralised if tests are to be run in automation. 
+
+To address these challenges, Galasa uses a Configuration Property Store (CPS) as part of the core Galasa framework. The CPS holds all the configuration values, so that the Galasa framework, Ecosystem, Managers, and individual tests can use the CPS to retrieve configuration information.
+
+A video -  <a href="https://www.youtube.com/watch?v=d_mufWVa31U" target="_blank">The Galasa Configuration Property Store </a> - is available on the Galasa YouTube channel. Watch the video to find out more about the features of the Galasa CPS, and to see a demonstration on how to manipulate CPS property values by using the Galasa command line tool.
+
+## Simplifying test configuration with the CPS
+
+When running Galasa tests locally, using the command line tool or Eclipse, the CPS is stored in the flat file `{GALASA_HOME}/cps.properties`. 
+
+When running tests in the Galasa Ecosystem, where tests are run on a Kubernetes cluster, a REST service hosts the storage of the CPS properties in a central location. A centralised CPS means that configuration information can be supplied to all automated tests running in a Galasa Ecosystem, allowing multiple tests to run using the same shared configuration information. 
+
+You can set, retrieve and delete properties that are held in the CPS by using the `galasasctl properties` command in the Galasa CLI. The ability to manage these properties directly makes it easier for testers to set parameters and credentials on the Ecosystem for tests to read and use at runtime. System administrators can use the CLI to set Ecosystem-wide configuration properties after Ecosystem installation.
 
 
 ## About the Configuration Properties Store 
 
+The CPS is hosted in the Ecosystem's etcd server, a key-value pair store which also hosts the Dynamic Status Store (DSS) and the Credentials Store (CREDs), maintaining a single source of truth about the status of the Ecosystem. For more information see the [Ecosystem Architecture](/docs/ecosystem/architecture) documentation.
+
 Properties in the CPS are dot separated values with lower and upper-case segments that define, for example, endpoint, port, and repository properties. These properties instruct the way in which a Galasa test runs.  It is the CPS and the configurational properties that enable tests to run against multiple environments, without changing the code inside the test. 
```

Comment:
> dot separated
should that be hyphenated ?

> These properties instruct the way in which a Galasa test runs.
That wording seems a bit odd ?
"These properties control how a Galasa test runs" ? is that better ?

Reply:
ok have updated